Global search enhancements #1738 (#1753)
SAP/luigi: docs/navigation-parameters-reference.md

    - **type**: Function
    - **description**: will be executed when the user clicks on a `searchResultItem`. 
    - **attribute** [searchResultItem](luigi-core-api.md#globalsearch)

>   - **inputPlaceHolder**
>     - **type**: string
>     - **description**: This text will be used as placeholder in the search input field. 
>    - **inputPlaceHolder**
>      - **type**: Function
>      - **description**: Output of this function will be used as placeholder in the search input field.
>    - **inputPlaceHolder**

Reviewer: ```suggestion
  - **inputPlaceholder**
    - **type**: string
    - **description**: This text will be used as placeholder in the search input field. 
   - **inputPlaceholder**
     - **type**: Function
     - **description**: Output of this function will be used as placeholder in the search input field.
   - **inputPlaceholder**
```

     - **type**: Object
     - **description**: Key value Json object where key is the language and value is the text used as placeholder in the search input field. This function doesn't have any input parameter;
   - **toggleSearch**
     - **type**: Function

>      - **description**: This function will be executed very time you click on search icon.

Reviewer 2: ```suggestion
     - **description**: This function will be executed every time you click on the search icon.
```

    - **attributes**:
      - **element**
        - **type**: INPUT element

>         - **description**: the input element where the user digit the search text.

Reviewer 2: ```suggestion
        - **description**: the input element where the user enters the search text.
```
I'm not entirely sure what you mean by "digit" here...
